### ISSUE TITLE
goimports: reports file is not `goimport`-ed on Windows

### ISSUE THREAD

**Issue by x1unix** [golangci/golangci-lint#635]
golangci-lint shows `goimport` warnings if file has `CRLF` line ending (on Windows).
`goimports` converts line endings back to `LF`, but on next `git pull` all line endings are converted back to `CRLF` (and makes a huge pain in ass to convert each file back or mess with git and IDE settings to use `LF` as line ending).

1. Version of golangci-lint: ` 1.17.1 built from 4ba2155 on 2019-06-10T09:06:49Z`
2. Config file: 
```yaml
run:
  # timeout for analysis, e.g. 30s, 5m, default is 1m
  deadline: 1500s

  # which dirs to skip: they won't be analyzed;
  # can use regexp here: generated.*, regexp is applied on full path;
  # default value is empty list, but next dirs are always skipped independently
  # from this option's value:
  #   	vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
  skip-dirs:
    - mock
  
  output:
    format: checkstyle

linters:
  enable-all: true
  disable:
    - gochecknoglobals
    - gochecknoinits
    - gocritic
    - lll
  
linters-settings:
  govet:
    check-shadowing: true
    settings:
      printf:
        funcs:
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Infof
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
  gocyclo:
    # minimal code complexity to report, 30 by default (but we recommend 10-20)
    min-complexity: 10
  maligned:
    suggest-new: true
  goconst:
    min-len: 2
    min-occurrences: 2
  misspell:
    locale: US
  goimports:
    local-prefixes: github.com/golangci/golangci-lint
  gocritic:
    enabled-tags:
      - performance
      - style
      - experimental
    disabled-checks:
      - wrapperFunc

issues:
  exclude-rules:
    - path: _test\.go
      linters:
        - gocyclo
        - errcheck
        - dupl
        - gosec
    - text: "weak cryptographic primitive"
      linters:
        - gosec
    - text: "G302"
      linters:
        - gosec
```
3. Go environment:

```
go version go1.12.7 windows/amd64
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\root\AppData\Local\go-build
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOOS=windows
set GOPATH=C:\Users\root\go
set GOPROXY=
set GORACE=
set GOROOT=C:\Go
set GOTMPDIR=
set GOTOOLDIR=C:\Go\pkg\tool\windows_amd64
set GCCGO=gccgo
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fmessage-length=0 -fdebug-prefix-map=C:\Users\root\AppData\Local\Temp\go-build756993502=/tmp/go-build -gno-record-gcc-switches
```
4. Verbose output of running: `golangci-lint run -v`

**Comment by jirfag** [comment]
Please, make an issue for goimports (x/tools repo) and we will update goimports
